<ui version="4.0" >
 <class>Osd</class>
 <widget class="QWidget" name="Osd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>100</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout" >
     <item>
      <widget class="QLCDNumber" name="channelNumber" >
       <property name="minimumSize" >
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>70</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="numDigits" >
        <number>3</number>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="value" stdset="0" >
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPlay" >
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Play</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStop" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Stop</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Back</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/player_rew.png</normaloff>:/icons/images/player_rew.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Next</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/player_fwd.png</normaloff>:/icons/images/player_fwd.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRecord" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Record Now</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/record.png</normaloff>:/icons/images/record.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMute" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Mute</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/icons/images/player_mute.png</normaloff>:/icons/images/player_mute.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum" >
        <number>200</number>
       </property>
       <property name="value" >
        <number>50</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelVolume" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>50</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="durationLayout" >
     <item>
      <widget class="QLabel" name="labelDuration" >
       <property name="text" >
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="durationSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLenght" >
       <property name="text" >
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>volumeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelVolume</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>584</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
